GoogleForCreators/web-stories-wp
# Editor: Update rogue Roboto to the Google Sans

## Context
Part of editor redesign.

## Summary
Attempts to find roboto usage still in the editor and update to google sans. Check out changed files for places I found it. Let me know if I missed and stragglers

## Relevant Technical Choices
NA

## To-do
If there's any I missed, update those.

## User-facing changes
Changes the font to roboto in the following places:
<img width="268" alt="Screen Shot 2021-03-30 at 2 47 37 PM" src="https://user-images.githubusercontent.com/35983235/113054771-09fa7500-9167-11eb-8919-ce5e537f8cac.png">
<img width="309" alt="Screen Shot 2021-03-30 at 2 47 29 PM" src="https://user-images.githubusercontent.com/35983235/113054777-0a930b80-9167-11eb-9f47-84c626f0730e.png">
<img width="365" alt="Screen Shot 2021-03-30 at 2 47 23 PM" src="https://user-images.githubusercontent.com/35983235/113054779-0a930b80-9167-11eb-9fe1-1fb9b65ce962.png">
<img width="641" alt="Screen Shot 2021-03-30 at 2 49 42 PM" src="https://user-images.githubusercontent.com/35983235/113054903-2f877e80-9167-11eb-9182-6d25a4267d8c.png">

Also updates the default text added from either the presets or the add text quick action to be google sans

## Testing Instructions
Double checkk the places in the editor I screen shotted above and make sure they're google sans. Also see that there's no other non google sans text in the editor




### QA
Follow testing instructions above





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6386
